### PR TITLE
fix: Handle invalid login credentials

### LINKED ISSUE #163: bug: Handle invalid login credentials
**Summary:**

Infinite progress bar when invalid login credentials are entered.

**Steps to reproduce:**

1. Enter invalid login credentials.
2. Infinite progress bar is shown.

**Expected behaviour:**
If invalid login credentials are entered, the progress bar is hidden and a message is shown.

**Observed behaviour:**
If invalid login credentials are entered, the progress bar is shown continuously.
ezgif com-video-to-gif (6)

**Device and Android version:**
Redmi Note 5 Pro - Android 9 (Pie)

### PR DESCRIPTION
**Pull request by prajwal27**
Fixes #163 

**Summary**
Hide progress bar after invalid wrong credentials entered.

**Observed Behaviour**
ezgif com-video-to-gif (6)

**Expected Behaviour**
ezgif com-video-to-gif (4)

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by rachittt** [comment]
@prajwal27 can you please resolve the conflict?

File: app/src/main/kotlin/org/mifos/mobile/cn/ui/utils/Toaster.kt [hunk 1]
```diff
@@ -28,6 +28,7 @@ object Toaster {
         textView.maxLines = 5
         snackbar.setAction("OK") { snackbar.dismiss() }
         snackbar.show()
+        hideSnackbad =  snackbar
```
Comment (rachittt) [inline app/src/main/kotlin/org/mifos/mobile/cn/ui/utils/Toaster.kt:31]:
```suggestion
        hideSnackbar =  snackbar
```

**Comment by rachittt** [comment]
Closing this because of inactivity.

**Comment by prajwal27** [comment]
can this be reopened?

**Pull request closed without merging**